docs/docs/advanced_user_topics/schema_inheritance.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Modeling Data Structure Inheritance
+parent: Advanced User Topics
+permalink: docs/advanced_user_topics/schema_inheritance
+---
+
+# Inheritance in AaC
+Architecture-as-Code provides users with the ability to model inheritance in their data structures in a limited capacity. Inheritance can only leveraged with `schema` definitions -- the definition used to define data structures.
+
+In order to define inheritance in AaC, a `schema` definition must populate the field `inherits` with a list of names of other `schema` definitions. Those definitions referenced in the `inherits` field will provide the inheriting definition with transitive attributes consisting of the parent definitions' `fields` and `validations` entries.
+
+## Example Usage
+Schema definitions inherit fields and validation rules from their parent definitions. So, if you were to define a required field in the parent definition, it will also be required in the child definition.
```

Comment:
```suggestion
Schema definitions inherit `fields` and `validation` rules from their parent definitions. So, if you were to define a required field in the parent definition, it will also be required in the child definition.
```